src/processing/script/transformers/index.ts

```diff
@@ -17,26 +18,28 @@ import windowPostMessageGetTransformer from './window-post-message-get';
 import postMessageCallApplyTransformer from './post-message-call-apply-bind';
 import forInTransformer from './for-in';
 import locationGetTransformer from './location-get';
-import locationPropertyGetTransformer from './location-property-get';
 import locationSetTransformer from './location-set';
 import propertyGetTransformer from './property-get';
 import propertySetTransformer from './property-set';
 import methodCallTransformer from './method-call';
 import jsProtocolLastExpression from './js-protocol-last-expression';
 import staticImportTransformer from './static-import';
 import dynamicImportTransformer from './dynamic-import';
+import declarationDestructuring from './declaration-destructuring';
+import assignmentDestructuring from './assignment-destructuring';
 
 export interface Transformer<C extends Node> {
     nodeReplacementRequireTransform: boolean;
     nodeTypes: C['type'];
     condition: (node: C, parent?: Node) => boolean;
-    run: <P extends Node>(node: C, parent?: P, key?: keyof P) => Node|null;
+    run: <P extends Node>(node: C, parent?: P, key?: keyof P, tempVars?: TempVariables) => Node|null;
```

Comment:
Do we will remove `__set$temp` and fix case for the nested `for-in` in separate PRs?

Reply:
Yes, I will do it in separate PRs.